client/my-sites/themes/theme-showcase.jsx

```diff
@@ -159,7 +159,10 @@ class ThemeShowcase extends Component {
 
 	scrollToSearchInput = () => {
 		if ( ! this.props.loggedOutComponent && this.scrollRef && this.scrollRef.current ) {
+			const yOffset = -55; // A number that takes the Adminbar as well as the screen options into account on mobile view ports.
```

Comment:
Ideally, we could dynamically take the height of these elements and calculate it before passing it here. That'll ensure this continues to work even if we do some style modifications in the future.

Reply:
Great point